app/src/main/java/com/example/songme/data/source/remote/ResponceFromAPI/DataResponseHandler.kt

```diff
@@ -0,0 +1,8 @@
+package com.example.songme.data.source.remote.ResponceFromAPI
+
+import com.example.songme.data.model.Track
+
+interface DataResponseHandler {
+    fun jsonToListTrack(jsonData: String): ArrayList<Track>
+    fun getResponses(url: String): ArrayList<Track>
```

Comment:
getResponse thôi chứ nhỉ 
```suggestion
    fun jsonToListTrack(jsonData: String): List<Track>
    fun getResponses(url: String): List<Track>
```